provisioning/local/gcp_kms.tf

```diff
@@ -0,0 +1,23 @@
+resource "google_kms_key_ring" "object_encryptor_keyring" {
+  name     = "${var.name_prefix}-object-encryptor"
+  location = "global"
```

Comment:
Ten `global` ma nejaky spesl vyznam - jako ze treba to stejne je nakonfigurovany v produkci? Nebo je to proste jen proto, aby si nad tim nemusel premyslet?

Reply:
abych nad tim nemusel přemýšlet, nezdá se, že by to na něco mělo vliv pro tenhle case